README.md

```diff
@@ -222,3 +223,10 @@ This only adds a plugin configuration file.
       config       => ['env.rootdn cn=admin,dc=example,dc=org'],
       config_label => 'slapd_*',
     }
+
+# Test
+
+Requirements: [Bundler](http://bundler.io/)
+
+    bundle install
```

Comment:
The defaults of "bundle install" is not safe. Running "bundle install" without extra options uses "sudo" transparently, and installs ruby gems into the library path of your system ruby. This may introduce incompatible changes for your packaged and installed applications.

Safer bundle invocation: "bundle install --path=vendor" to create a "vendor" directory in the current directory, or "bundle install --path=~/.cache/bundler/puppet-mod-tests" to share a directory for the ruby gems with other puppet modules.  "bundle exec" will read ".bundle/config" in the current directory, or any path above it, to find the path used.